### ISSUE TITLE
[Accessibility issue] Vignettes lacks alt= in plots

### ISSUE THREAD

**Issue by MichaelChirico** [mfasiolo/mvnfast#13]
We have a tool that flags images that lack an `alt=` attribute. See https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices -- these are essential for conveying the image contents to users on screen readers.

You can add these to your vignettes by using the recent-ish `fig.alt` chunk option in `knitr`; see https://www.rstudio.com/blog/knitr-fig-alt/.

The vignette at issue is https://github.com/mfasiolo/mvnfast/blob/32e06e8a014716d832c908ea2c5d689f9d620bbe/vignettes/mvnfast.Rmd.

Unfortunately the lines are flagged in the HTML output, which won't correspond exactly to the .Rmd file lines, but I assume if you're familiar with the vignette it will be clear which images need it:

```
insight/inst/doc/mvnfast.html:382
```

**Comment by mfasiolo** [comment]
Ok I'll make sure to add the appropriate fig.alt options to the vignette. Thanks for the pointers!
